src/MeshIntegrals.jl

```diff
@@ -1,7 +1,7 @@
 module MeshIntegrals
 using CliffordNumbers: CliffordNumbers, VGA, ∧
 using CoordRefSystems: CoordRefSystems, CRS
-using Meshes: Meshes, Geometry
+using Meshes: Meshes, Geometry, to
```

Comment:
```suggestion
using Meshes: Meshes, Geometry
```
This would not be necessary anymore with the suggestion above.